Lib/fontTools/varLib/__init__.py

```diff
@@ -778,6 +778,7 @@ def main(args=None):
 	parser.add_argument('designspace')
 	parser.add_argument('-o', metavar='OUTPUTFILE', dest='outfile', default=None, help='output file')
 	parser.add_argument('-x', metavar='TAG', dest='exclude', action='append', default=[], help='exclude table')
+	parser.add_argument('--disableIUP', dest='optimize', action='store_false', default=True, help='do not perform IUP optimization')
```

Comment:
`action='store_false'` defaults to True (https://docs.python.org/3/library/argparse.html#action), setting `default=True` is not necessary.

Reply:
:bike: :house_with_garden:  I don't like the camelCase `--disableIUP` too much. Maybe `--disable-IUP`?

Reply:
`--disable-iup` would even be better, upper case in command line options looks unusual.